src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -527,6 +527,10 @@ def load_arguments(self, _):
             c.argument('nodepool_id', required=True, validator=validate_nodepool_id, help='The nodepool id.')
             c.argument('aks_custom_headers')
 
+    with self.argument_context('aks nodepool snapshot update') as c:
+        c.argument('snapshot_name', options_list=['--name', '-n'], required=True, help='The nodepool snapshot name.', validator=validate_snapshot_name)
+        c.argument('tags', tags_type, required=True, help='The tags to set to the snapshot.')
```

Comment:
```suggestion
        c.argument('snapshot_name', options_list=['--name', '-n'], help='The nodepool snapshot name.', validator=validate_snapshot_name)
        c.argument('tags', tags_type, help='The tags to set to the snapshot.')
```
Since the corresponding parameters in the custom method do not have default values, the default setting for these parameters is `required=True`, there is no need to explicitly define them repeatedly